src/loaders.rs

```diff
@@ -49,6 +49,7 @@ pub fn load_file(path: impl AsRef<Path>) -> Result<Model, LoaderError> {
     match path.extension().and_then(OsStr::to_str) {
         Some("obj") => obj::load_file(path).map_err(Into::into),
         Some("gltf") => gltf::load_file(path).map_err(Into::into),
+        Some("glb") => gltf::load_file(path).map_err(Into::into),
```

Comment:
You can combine these two cases using `Some("gltf") | Some("glb")` as the pattern. Once that is fixed this is good to go. Thanks!

Reply:
sick